build.sbt

```diff
@@ -173,6 +173,17 @@ lazy val io = project.in(file("io")).
     name := "fs2-io"
   ).dependsOn(coreJVM % "compile->compile;test->test")
 
+lazy val scodec = crossProject.in(file("scodec")).
```

Comment:
Not part of scodec-stream because it only depends on scodec-bits, and because scodec-core is a faster moving target.  Maybe the name should be more specific. :shrug: